channeldb/invoices.go

```diff
@@ -21,6 +21,13 @@ var (
 	// preimage for this invoice is not yet known.
 	unknownPreimage lntypes.Preimage
 
+	// LegacyKeysendPayAddr is a sentinel payment address for legacy keysend
+	// payments. Invoices with this payment address are special-cased in the
+	// insertion logic to prevent being indexed in the payment address
+	// index, otherwise they would cause collisions after the first
+	// insertion.
+	LegacyKeysendPayAddr [32]byte
```

Comment:
It is also the sentinel value for legacy non-keysend invoices. Currently it is only used for `AddInvoice`, so the variable name is correct, but if we start using this elsewhere a more generic name would be better.

Reply:
fixed